### ISSUE TITLE
Evaluation mode

### ISSUE THREAD

**Issue by GabrielDornelles** [GabrielDornelles/MK7-AutoPilot#1]
```py
model.eval()
```
is never called in run.py.

 There's no Dropout or Batchnorm layers in the model, but I wonder if model would infer different if eval mode is set.
